config/initializers/voyager_helpers.rb

```diff
@@ -1,5 +1,4 @@
 require 'yaml'
-require_relative '../../voyager_helpers/lib/voyager_helpers'
 include VoyagerHelpers::OracleConnection
 
 vh_config = YAML.load_file("#{File.dirname(__FILE__)}/../voyager_helpers.yml")
```

Comment:
Standalone usage of voyager_helpers is going to need this too, right?  So maybe the creation of this initializer (or at least docs about doing that) should be in there?